server/game/cards/characters/03/wolvesofthenorth.js

```diff
@@ -4,22 +4,19 @@ class WolvesOfTheNorth extends DrawCard {
     setupCardAbilities() {
         this.reaction({
             when: {
-                onBypassedByStealth: (event, challenge, source, target) => {
-                    if(source !== this) {
-                        return false;
-                    }
-
-                    this.bypassed = target;
-
-                    return true;
-                }
+                onChallengeInitiated: event => event.challenge.isStealthSource(this)
             },
-            handler: () => {
-                this.game.addMessage('{0} uses {1} to give {2} a penalty to STR', this.controller, this, this.bypassed);
+            handler: context => {
+                let target = context.event.challenge.getStealthTargetFor(this);
+                let strDecrease = -this.controller.getNumberOfCardsInPlay(c => c.hasTrait('Direwolf'));
+
                 this.untilEndOfPhase(ability => ({
-                    match: this.bypassed,
-                    effect: ability.effects.dynamicStrength(() => -this.controller.getNumberOfCardsInPlay(c => c.hasTrait('Direwolf')))
+                    match: target,
+                    effect: ability.effects.modifyStrength(strDecrease)
```

Comment:
Also good catch - value should be fixed at the time of the ability, not dynamic. 👍